databricks/koalas/frame.py

```diff
@@ -691,8 +727,88 @@ def loc(self):
     def copy(self):
         return DataFrame(self._sdf, self._metadata.copy())
 
-    @derived_from(pd.DataFrame)
     def dropna(self, axis=0, how='any', thresh=None, subset=None, inplace=False):
+        """
+        Remove missing values.
+
+        Parameters
+        ----------
+        axis : {0 or 'index', 1 or 'columns'}, default 0
```

Comment:
I think we don;t support `columns` yet. Let's take out.

Reply:
Doing it.